assets/blocks/quiz/quiz.editor.scss

```diff
@@ -50,4 +52,19 @@ $gray-400: #ccc;
 			padding: 6px 24px;
 		}
 	}
+
+	&__appender {
+
+		text-align: center;
+
+		.editor-styles-wrapper &__placeholder {
+			opacity: 0.62;
+			margin: 0;
+			margin-top: -12px;
```

Comment:
This is just a very tiny nit, but I wonder if this could be tiny bit higher (i.e.`-8`?). Tiny bit squished. I'm glad we're adding this helper message, though. 

<img width="240" alt="Screen Shot 2021-03-04 at 10 36 36 pm" src="https://user-images.githubusercontent.com/68693/110040057-16430c00-7d3a-11eb-99ea-16bc80f34cd4.png">

Reply:
Cleaned this up in 05ac50e